__tests__/index.js

```diff
@@ -39,7 +39,7 @@ describe('DatadogTransport#log(info, callback)', () => {
     }
   ]
     .forEach(testCase => {
-      it(testCase.case, async () => {
+      it(testCase.case, callback => {
```

Comment:
Why though?

Reply:
See https://github.com/winstonjs/winston/issues/1532. Winston transports do not support async log functions

This appears to work fine as-is for the most part, but in corner cases where the program may be terminated soon after trying to send a log, you are not currently able to rely on winston's documented feature of a 'finish' event being emitted (I suspect because the callback is immediately executed).

Reply:
This specific line is a jest feature to allow the jest test execution to wait until the callback is executed (or timeout and fail the test). It appears to be the recommended way to test callback based functions, and this was also required to be able to test the new function (without `async`/`await`)